cli/tsc/dts/lib.deno.ns.d.ts

```diff
@@ -1208,6 +1208,14 @@ declare namespace Deno {
     end(): void;
   }
 
+/** Interface for serving HTTP requests. */
+interface ServeDefaultExport {
```

Comment:
Looks like we got two `ServeDefaultExport` now :D can you please move `onListen` declaration to the existing interface on line 5068?

Reply:
Will do, working on it ASAP